A39-xds-http-filters.md

```diff
@@ -233,9 +236,9 @@ gRPC will support the [router
 filter](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/router_filter),
 which is used for the proto message type
 [`envoy.extensions.filters.http.router.v3.Router`](https://github.com/envoyproxy/envoy/blob/18db4c90e3295fb2c39bfc7b2ce641cfd6c3fbed/api/envoy/extensions/filters/http/router/v3/router.proto#L23).
-In practice, in order for gRPC to function properly, this filter is
-required and must be last in the filter list, although that is not
-enforced at config validation time.
+This filter is a terminal filter, so it must be the last one in the
```

Comment:
This isn't strictly true given the above. "A" terminal filter must be last, but it doesn't have to be this one. I think this needs to be reworded if this is the only supported terminal filter.

Reply:
Done.